uwp/Pdf-Viewer/Concepts-and-Features/Working-with-free-text-callout-annotation.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: Free text calllout annotation in UWP PDF Viewer control | Syncfusion
+description: Learn here all about free text callout annotation support in Syncfusion UWP PDF Viewer (SfPdfViewer) control and more.
+platform: UWP
+control: SfPdfViewerControl
+documentation: ug
+---
+
+# Working with free text callout annotations in UWP PDF Viewer (SfPdfViewer)
+
+PDF Viewer allows you to include free text callout annotations in a PDF document and provides options to modify or remove the existing free text callout annotations.
```

Comment:
The PDF Viewer allows you to include free text callout annotations in a PDF document and provides options to modify or remove the existing free text callout annotations.